### DISCUSSION TITLE
Is it possible to remove GF from the bg?

### DISCUSSION THREAD

**Discussion by libracrumbs** [KadeArchive/Kade-Engine#1854] [Q&A] [answered]
Title says it all

**Comment by SuxxorState** [comment, marked as answer]
yes, there are two solutions for this, both require you to know how to compile and decompile the game(so just kinda warning ya ig???). in the file PlayState.hx, navigate down to around line 953, there you will find where the gf is spawned. you can either a. move gf so she's offscreen, or b. delete that entire chunk of code(havent tested whether that works so it may not). ultimately though if you want it to be song-specific id recommend using a case function and just moving her offscreen.

**Reply by SuxxorState** [reply]
oh, and you can just, ya know, make the GF_Assets a blank image. that works too.

**Reply by libracrumbs** [reply]
how can i make another gf spritesheet?

**Reply by SuxxorState** [reply]
first, clone the original gf sheet. name it to whatever you want. with it clone the .xml file and name it to the same thing. go into the .xml file and change the texture atlas at the very top to lead to the png. next, go into the characters.hx file. scroll down to around line 36, where you will find the girlfriend case. select around to where it says playAnim('danceRight');, this should be around 20 lines worth of code. copy said code and paste it underneath the original. then, change the case in this copied code to what you want to name your void of a gf. then, go to the next line and change the texture file to the name of your png and xml (do not include .png or .xml). next, find the gfversionlist.txt file (located in preload/data) and add your gf (this is where the name you chose for the 'case' function comes in). finally, compile the game and select from the debug menu

**Reply by libracrumbs** [reply]
tysm!

**Comment by SuxxorState** [comment]
![gf](https://user-images.githubusercontent.com/55562463/130335920-5f40d01c-e11d-43c0-9fc1-1f1d2aa7471f.PNG)
here's around what you need to select, it wont look exactly the same as this but it will look similar
